node/src/chain_spec/mod.rs

```diff
@@ -0,0 +1,109 @@
+use serde::{Deserialize, Serialize};
```

Comment:
it's a rust lib format thing. When you create a folder like this you need to create a mod file. This will expose the contents of the folder via an import of the folder name